spinalcordtoolbox/deepseg/models.py

```diff
@@ -126,7 +127,33 @@
         "description": "Lumbar SC segmentation on T2w contrast with 3D UNet",
         "contrasts": ["t2"],
         "default": False,
-    }
+    },
+    "model_seg_sc_contrast_agnostic_softseg_nnunet": {
+        "url": [
+            "https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/releases/download/v2.0/model_2023-09-18.zip"
+        ],
+        "description": "Spinal cord segmentation that is agnostic to contrast using NNUnet",
+        "contrasts": ["any"],
+        "default": False,
+    },
+    "model_seg_sci_multiclass_sc_lesion_nnunet": {
+        "url": [
+            # TODO: Replace with official release (*not* from joshuacwnewton)
+            "https://github.com/joshuacwnewton/model_seg_sci/releases/download/r20230814/model-sci-multisite_r20230814.zip"
+        ],
+        "description": "Traumatic SCI spinal cord/lesion segmentation for T2w contrast",
+        "contrasts": ["t2"],
+        "default": False,
+    },
+    "model_seg_spinal_rootlets_nnunet": {
+        "url": [
+            # TODO: Replace with official release (*not* from joshuacwnewton)
+            "https://github.com/joshuacwnewton/model-spinal-rootlets/releases/download/r20230815/model-spinal-rootlets_r20230815.zip"
+        ],
```

Comment:
Note to self: This URL will need to be replaced once an official release on the main repo has been made.

Reply:
Updated! :)

Reply:
With the new model, the rootlets predictions are quite different:

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/33d7fd7d-9322-43e9-91e6-139d10970cbc)

Hence the failing tests. But, that's mainly because I was comparing the _exact_ coordinates of the detected labels. It's probably simpler to just check against the label values, which will make the test a bit more lax, but still ensure that the correct label values are output, which feels like a "good enough" test for a black box model.